sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveContext.scala

```diff
@@ -117,6 +117,11 @@ class HiveContext(sc: SparkContext) extends SQLContext(sc) {
   }
 
   /**
+   * Returns the list of tables for this SQLContext
+   */
+  def showTables(databaseName: Option[String]): Seq[String] = catalog.showTables(databaseName)
```

Comment:
Can we define two functions rather than having an option?

i.e.

def showTables(): Array[String]

and

def showTables(databaseName: String): Array[String]